docs/usage.md

```diff
@@ -74,9 +74,9 @@ $ elastic-agent-changelog-tool cleanup
 ```
 $ elastic-agent-changelog-tool render --version x.y.z
 ```
-* This will generate an asciidoc fragment in __TODO__.
+* This will generate an asciidoc fragment in the current directory.
```

Comment:
There have been a slight change to help with consistency in the latest commits, this should be changed in:

```suggestion
* This will generate an asciidoc fragment in the `changelog/` directory.
```